spanner/api/Spanner.Samples.Tests/DeleteInstanceConfigAsyncTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Spanner.Samples.Tests
+{
+    [Collection(nameof(SpannerFixture))]
+    public class DeleteInstanceConfigAsyncTest
+    {
+        private readonly SpannerFixture _spannerFixture;
+
+        public DeleteInstanceConfigAsyncTest(SpannerFixture spannerFixture)
+        {
+            _spannerFixture = spannerFixture;
+        }
+
+        [Fact]
+        public async Task TestDeleteInstanceConfigAsync()
+        {
+            DeleteInstanceConfigAsyncSample deleteInstanceConfigAsyncSample = new DeleteInstanceConfigAsyncSample();
+            await deleteInstanceConfigAsyncSample.DeleteInstanceConfigAsync(_spannerFixture.ProjectId, _spannerFixture.CustomInstanceConfigId);
```

Comment:
If the delete config test runs before the update config test, then the update config test won't work, i.e. won't do anything I know it won't fail because you have a try catch but still we won't test anything.

I prefer if you create a temporary config here to use for deletion.